### DISCUSSION TITLE
想请问一下Termux里安装的alist什么时候才能更新3.37.3

### DISCUSSION THREAD

**Discussion by sweetbearbear** [AlistGo/alist#7237] [Q&A]
如题，现在还是3.37.1，115网盘加载不出来，是不是需要等几天才可以更新最新版🥲

**Comment by nxjonson** [comment]
在这里直接下载对应的android版本，解压覆盖alist文件即可在termux下运行。

**Reply by goodok3000** [reply]
很感谢你做了回复，我有同样的问题。理解你说的，但不会操作，也在网上找了，没找到相应的内容。现在能做到
wget 相应版本的android的alist-android-arm64.tar.gz，然后解压缩tar -zxvf alist-android-arm64.tar.gz ，得到alist。再往下实在不会操作了。不管怎么折腾，Termux用pkg install alist ,安装的都是镜像里的3.37.1版本。
请你，指教一下如何覆盖掉镜像里的alist安装文件。请给具体操作命令行。拜托了，困扰了好几天。
谢谢！

**Reply by nxjonson** [reply]
请看看/data/data/com.termux/files/usr/bin/目录下是否有alist二进制文件，覆盖它即可。

**Reply by goodok3000** [reply]
/data/data/com.termux/files/usr/bin/ 这目录下安装alist之前没有alist文件，pkg install allist之后才有，然后用下载的alist覆盖它，执行这个pkg install alist还是从镜像取文件安装 仍然是3.37.1版本 。既然提供了本地文件，肯定是能安装，但是没说明文件，我知识太少，解决不了。辛苦了！谢谢！查了一下，没找到如何安装本地文件。

**Reply by nxjonson** [reply]
覆盖后，退回到termux，执行 alist server就可以正常进入alist了。如果执行pkg install alist，就是走了回头路……

**Reply by goodok3000** [reply]
好 ，我试试。谢谢！

**Reply by goodok3000** [reply]
谢谢高人nxjonson！明白了，完全解决了！你不出手，我是没办法解决的。
